docs/UserGuide.adoc

```diff
@@ -224,7 +224,7 @@ Edits the name of the borrower to be `Betsy Crower`
 
 Loan book(s) by their serial number +
 Format: `loan sn/BOOK_SN` +
-Format: `loan sn/BOOK_SN [sn/BOOK_SN]...` - Coming in v2.0
```

Comment:
I think this is as such because we want at least 1 serial number, while the [sn/BOOK_SN] notation will allow 0 counts of it, which is wrong in this case. So we add a mandatory sn/BOOK_SN in front, followed by optional counts of [sn/BOOK_SN]

Reply:
Funny. I didn't touch this line. Probably a badly resolved merge conflict. Thanks for the catch!